scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime, timedelta
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+m = DuplicateModel.load("duplicatemodel")
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+test_bugs = bugzilla.download_bugs_between(
+    datetime.now() - timedelta(days=21), datetime.now(), store=False
+)
+
+test_bugs = [bug for bug in test_bugs if not bug["creator"] in REPORTERS_TO_IGNORE]
+
+bug_tuples = itertools.combinations(test_bugs, 2)
+
+# Warning: Classifying all the test bugs takes a while
+probs = [m.classify(bug_tuples)]
```

Comment:
m.classify should already be returning a list, so if you do this you'll get a list with a single element being a list of predictions.
Also, you should use probabilities=True, or you won't get probabilities.